gcloud-java-storage/src/main/java/com/google/gcloud/storage/BlobReadChannelImpl.java

```diff
@@ -59,19 +59,18 @@
     initTransients();
```

Comment:
I don't think this is no longer needed. could be set in the constructor (and both members can be final).